docs/agents/config.md

```diff
@@ -0,0 +1,184 @@
+# Configuration Guide
+
+## Overview
+
+Quay configuration is managed through `config.yaml` (runtime) and validated by `config-tool/` (Go).
+
+| File | Purpose |
+|------|---------|
+| `config.py` | Python config defaults (customer-facing) |
+| `features/__init__.py` | Feature flag loading |
+| `config-tool/pkg/lib/fieldgroups/` | Go validation schemas |
+| `util/config/schema.py` | Schema docs and INTERNAL_ONLY_PROPERTIES |
+
+## Config Categories
+
+Quay has three categories of configuration:
+
+| Category | In DefaultConfig? | When to Use |
+|----------|-------------------|-------------|
+| Customer-facing | Yes | Downstream Quay deployments, documented |
+| Internal-only | Yes (marked `[QUAY.IO]`) | In DefaultConfig but hidden from customer docs |
+| Runtime-only | No | Quay.io infrastructure, accessed via `app.config.get()` |
```

Comment:
Again not correct, for every single feature we use `app.config.get()` because that loads that feature along with its settings from the `config.yaml` file.